### PR TITLE
Fit nextcloud css guidelines

### PR DESCRIPTION
**Pull request by skjnldsv**
Should probably be best with the last fix from nextcloud/server#3024

### CODE REVIEW

File: js/templates/account.html [hunk 1]
```diff
@@ -13,8 +13,6 @@ <h2 class="mail-account-email" title="{{emailAddress}}">{{emailAddress}}</h2>
 <div class="app-navigation-entry-menu">
 	<ul>
 		<li><button class="icon-rename svg" title="{{ t 'Settings' }}"></button></li>
-	</ul>
```
Comment (ChristophWurst) [inline js/templates/account.html]: 🙈

**Comment by ChristophWurst** [comment]
bildschirmfoto von 2017-01-12 09-41-39
 looks strange on FF

File: js/templates/account.html (hunk 1 above)
Comment (skjnldsv) [inline js/templates/account.html]: ?

Comment (ChristophWurst) [inline js/templates/account.html]: don't know why we had two list there

Comment (skjnldsv) [inline js/templates/account.html]: ^^

**Comment by skjnldsv** [comment]
@ChristophWurst do you mean the scrollbar?

**Comment by ChristophWurst** [comment]
yes ;-)

**Comment by skjnldsv** [comment]
Well it isn't related to this pr ^^
And the firefox scrollbar has always been awful 😱 
capture d ecran_2017-01-12_13-52-08

**Comment by ChristophWurst** [comment]
> Well it isn't related to this pr ^^

will check again later. IIRC that little scrollbar just scrolls the three dots, not the entire list of accounts/folders.

**Comment by ChristophWurst** [comment]
Still the same issue when I'm on https://github.com/nextcloud/server/pull/3024 and this PR's branch:
bildschirmfoto von 2017-01-12 23-02-36

**Comment by skjnldsv** [comment]
Fixed.

**Comment by ChristophWurst** [comment]
@skjnldsv which server branch did you use to test this? On master it works, but in combination with  nextcloud/server#3024 I still get the scroll bar :-/

**Comment by skjnldsv** [comment]
@ChristophWurst the last update commit of this pr: 83d078b6e4c6dd5c16bca72f7a58f30a169df12d
with the latest nextcloud/server#3024 at ce9c833c5b7b59f846bd36f08cc25ad06482a054 Sorry I forgot to push 😞

**Comment by skjnldsv** [comment]
😆

**Pull request merged**